src/ir_York.h

```diff
@@ -0,0 +1,136 @@
+// Copyright 2022 Daniele Gobbetti
+
+/// @file
+/// @brief Support for the York AC protocol (remote GRYLH2A)
+
+// Supports:
+//   Brand: York,  Model: MHH07P17
```

Comment:
Specify it's an AC. (just a nitpick)
Is there a remote model number too? ;-) You should added that too.
e.g.
```c++
//   Brand: York,  Model: MHH07P17 A/C
//   Brand: York,  Model: GRYLH2A remote
```